### ISSUE TITLE
Old examples within `examples/drafts` are confusing. Move `how-tos` to top level of repo. Only keep working and tested examples.

### ISSUE THREAD

**Issue by crcrewso** [pymedphys/pymedphys#1704]
[tps_compare](https://github.com/pymedphys/pymedphys/blob/8c55ae66c182f7e268f235322b498fc1ac34eccf/examples/drafts/tps_compare.ipynb)

```python
import pymedphys.experimental.tpscompare
```

In the last line of the import init cell there is a reference to tpscompare. It looks like this code has been significantly refactored since this example was first written as this reference no longer exists.

**Comment by SimonBiggs** [comment]
Yeah, that tps_compare file is within `examples/drafts`. I expect we should potentially merge the concept of "how-tos" and "examples", and then have it so that the how-tos within the docs first copy from the root examples directory.

Then, we clear house on old deprecated examples, and allow for more examples to be fleshed out that actually end up running through the CI suite (via the jupyterbook execution).
